### ISSUE TITLE
GIMP 2.10.12 and 2.10.14 crashes (session?) when opening an image

### ISSUE THREAD

**Issue by jaredyork** [snapcrafters/gimp#96]
**GIMP version:** 2.10.12 and 2.10.14
Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of GIMP or on updated development code
(master branch).
**Operating System:** Xubuntu (Xfce 4.12, distributed by Xubuntu)
**Package:** snap (normal channel and beta)

**Description of the bug**
Please describe your issue with details.
Add screenshot or other files if needed.
My session? crashes when I open an image with both GIMP 2.10.12 and GIMP 2.10.14.
I recorded a video of what I'm seeing: text

**Reproduction
Is the bug reproducible? [Always / Randomly / Happened only once ]
Reproduction steps:**

Open any image

…
**Expected result:**
The image should open
**Actual result:**
Everything freezes, screen goes black with cursor, then it usually makes me log back in.  None of my applications are open when I log back in.

After doing a bunch of research on how to produce a stack trace, the only relevant command I was able to find was: "snap run --strace="-tt -r" gimp 2>&1 | tee gimp_trace.log".  The resulting file is 18.5 MB, and I'm not sure if debugging symbols were enabled via the method I used.  Does this log file size sound about right?   There are many mentions of GIMP/the GIMP snap in the log.  Here is a [link to a GIMP 2.10.14 stack trace](https://www.mediafire.com/file/8hjje01x7j37k54/gimp_strace.log/file) to download this log file.  I tried the methods on Gnome wiki regarding [capturing stack traces](https://wiki.gnome.org/Community/GettingInTouch/Bugzilla/GettingTraces/Details) with no avail.  

Others have mentioned to install or compile debug symbols, is that some sort of system-wide package I need to find, or is it specifically built for snap/GIMP?  Or are the debug symbols already in the stack trace I linked?  Please forgive me for my lack of knowledge in this area; I do try to do a fair bit of searching before I ask questions.  I'm pretty new to generating stack traces, etc.

Thanks,
Jared

**Comment by jnsgruk** [comment]
Hi there! We recently merged a big overhaul of the GIMP snap, keeping us up to date with the latest 3.0 feature release.

The snap has been rebased on core24, has migrated to a much improved GNOME extension & content snap, and is now using many more components from the Ubuntu archive. Given either the version stated in this issue, or the last time the issue was updated, it's highly likely that the issue is resolved or is no longer relevant to the supported version of GIMP.

As part of getting the snap and the repo back on track, and to help us prioritise ongoing issues, I'm working through all issues that refer to older versions or haven't been updated in a while.

I'm closing this issue for now, but please comment if the issue still persists, and I'll reopen and prioritise getting it fixed, or at least get you an update!
